Sources/UniformTypeIdentifier/PublicTypes.swift

```diff
@@ -225,6 +228,31 @@ public extension UniformTypeIdentifier {
     static let oggApplication = UniformTypeIdentifier(mimeType: "application/ogg")!
 }
 
+public extension UniformTypeIdentifier {
+    static let googleDocument = UniformTypeIdentifier(
+        mimeType: "application/vnd.google-apps.document",
+        conformingTo: .compositeContent
+    )!
+    static let googleSlidesPresentation = UniformTypeIdentifier(
+        mimeType: "application/vnd.google-apps.presentation",
+        conformingTo: .presentation
+    )!
+    static let googleSheetsSpreadsheet = UniformTypeIdentifier(
+        mimeType: "application/vnd.google-apps.spreadsheet",
+        conformingTo: .spreadsheet
+    )!
+}
+
+public extension UniformTypeIdentifier {
+    static let affinityPhoto = UniformTypeIdentifier(fileExtension: "afphoto")!
+    static let affinityDesign = UniformTypeIdentifier(fileExtension: "afdesign")!
+    static let sketchFile = UniformTypeIdentifier(
+        fileExtension: "sketch",
+        conformingTo: .zipArchive
+    )!
+    static let adobeXd = UniformTypeIdentifier(fileExtension: "xd", conformingTo: .zipArchive)!
```

Comment:
They both has` application/zip` mime

Reply:
@sashkopotapov Can you add tests to make sure we can actually initialize the Affinity and Adobe UTIs?